src/main/java/frc/robot/utilities/pid/PIDFCoefs.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.utilities.pid;
+
+import com.google.gson.annotations.SerializedName;
+
+public class PIDFCoefs extends PIDCoefs {
+    @SerializedName(value = "kV", alternate = {"v", "V", "KV"})
+    private double KV;
+    @SerializedName(value = "kS", alternate = {"s", "S", "KS"})
+    private double KS;
+
```

Comment:
what about kA?

Reply:
we have had no use for kA ever. if you want me to add it so we can have a true simpleMotorFeedforward sure but we don't use the same math as WPI

Reply:
What do you mean, we don't use the same math? And are you trying to tell me that the acceleration is neglegable?